shims/spark310/src/main/scala/com/nvidia/spark/rapids/shims/spark310/Spark310Shims.scala

```diff
@@ -102,30 +102,7 @@ class Spark310Shims extends Spark301Shims {
   }
 
   override def getExprs: Map[Class[_ <: Expression], ExprRule[_ <: Expression]] = {
```

Comment:
you can just remove this entire thing because Spark310Shims extends Spark301Shims

Reply:
ok nevermind, i forgot that this essentially removes the GPUTimeSub